test/integration/truefi.test.ts

```diff
@@ -71,7 +71,7 @@ describe('TrueFi', () => {
       'joiningFee',
       'claimableFees',
       'fundsManager',
-      '_stakeToken',
+      // '_stakeToken',
```

Comment:
Out of curiosity, any reason we keep these old parameters commented instead of deleting them?

I guess it might help with figuring out what's changed between deploys/upgrades?